action.yaml

```diff
@@ -22,6 +27,7 @@ runs:
   using: composite
   steps:
     - uses: actions/checkout@v2
+      if: ${{ ! inputs.skip-checkout }}
```

Comment:
The `${{}}` is not needed here. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif.